app/src/main/contrib/components/Responsibilities/Overview/ResponsibilityOverviewPage.tsx

```diff
@@ -34,17 +34,17 @@ export class ResponsibilityOverviewPage extends PageWithHeaderAndNav<LocationPro
             On this page you can:
             <ol>
                 <li>
-                    See an overview of which scenarios your group are responsible for providing impact estimates for.
+                    See an overview of which scenarios your group is responsible for providing burden estimates for.
                     If we have the wrong scenarios listed, please contact us <a href={ supportEmail }>here</a>.
                 </li>
                 <li>Download demographic data which applies to all scenarios</li>
                 <li>Download coverage data for each scenario</li>
-                <li>Upload impact estimates for each scenario, and review any problems the system has detected in the
+                <li>Upload burden estimates for each scenario, and review any problems the system has detected in the
                     uploaded data.
                 </li>
-                <li>Track progress towards providing impact estimates for all your scenarios</li>
+                <li>Track progress towards providing burden estimates for all your scenarios</li>
             </ol>
-            <ResponsibilityList />
+            <ResponsibilityOverviewContent />
         </div>
```

Comment:
Updated the copy while I was here. Replaced "impact estimates" with "burden estimates".